lib/types/api.ts

```diff
@@ -77,7 +77,7 @@ export interface components {
       name: string;
       description?: string;
       /** Format: date-time */
-      created?: DateConstructor;
+      created?: string;
```

Comment:
Apparently I have managed to change the type of 'created' from string to DateConstructor due to some unknown reasons. I checked the master-branch and the type of 'created' was originally a string so I reverted it.